### DISCUSSION TITLE
CLI issue

### DISCUSSION THREAD

**Discussion by PauliusGitSoft** [immich-app/immich#5324] [Q&A]
Hello,
having issue with updated Immich CLI
what I am doing wrong?
![image](https://github.com/immich-app/immich/assets/145255012/4d4e17f6-3c45-47e3-ac5c-6adb1fd2c9e0)

**Comment by atomaka** [comment]
Include the API path in your server URL: `http://192.168.1.12:2283/api`

**Reply by PauliusGitSoft** [reply]
I tried first with /api but same issue.

**Reply by jrasm91** [reply]
Can you post a screenshot of this?

**Reply by PauliusGitSoft** [reply]
![image](https://github.com/immich-app/immich/assets/145255012/13b5dc1c-72a9-4856-bfec-db60f18eb094)
missed "sudo", now works thanks for help :)
